File: LinkedList/easy/9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int rev,temp;
        int sum=0;
        temp=x;
        while(x>0)
        {
            rev=x%10;
            sum=(sum*10)+rev;
            x=x/10;
        }
        if(sum==temp)
            return true;
        return false;
    }
}

File: LinkedList/easy/9-palindrome-number/palindrome-number.c
bool isPalindrome(int x){

    int temp, rev;
    long sum=0;

    temp=x;

    while(x>0)
    {
        rev=x%10;
        sum=(sum*10)+rev;
        x=x/10;
    }

    if(temp==sum)
        return 1;
     return 0;   
}